meta-oe/recipes-support/genimage/genimage_18.bb

```diff
@@ -0,0 +1,3 @@
+require genimage.inc
```

Comment:
.inc files are only needed when there are more than 1 recipes needing them so please merge it into .bb